debian/jessie/foreman/control

```diff
@@ -14,7 +14,7 @@ Section: web
 Priority: extra
 Depends: ruby2.1, bundler (>= 1.3), zlib1g-dev,
          ruby2.1-dev, build-essential,
-         rake (>=0.8.4), ${misc:Depends}
+         rake (>=0.8.4), ruby-foreman-tasks (>= 0.8.5), ${misc:Depends}
```

Comment:
Not sure this should be a separate dependency - it's likely to cause problems adding a circular dependency. The bundler.d file is already provided so it should just be vendored like other dependencies.